src/pages/admin-ui-sdk/extension-points/order.md

```diff
@@ -43,6 +43,71 @@ order: {
 }
 ```
 
+### Sample GraphQL Mesh configuration file
```

Comment:
```suggestion
### Sample API Mesh configuration file
```
The Order page should likely replace any references to "GraphQL Mesh" with "API Mesh", with the first mention containing the full name "API Mesh for Adobe Developer App Builder".